lib/email_repair/mechanic.rb

```diff
@@ -78,7 +78,7 @@ def self.common_domains
     class CommonDomainSuffixRepair < CommonDomainRepair
       def self.repair(email)
         common_domains.each do |name, suffix|
-          email = "#{email}.#{suffix}" if email.match(/#{name}$/)
+          email = "#{email}.#{suffix}" if email =~ /#{name}$/
```

Comment:
We haven't really discussed which versions of ruby we intend to support with this gem, but in `2.4` the `match?` predicate method was introduced, which I'm a fan of.

Reply:
since that would be a potentially breaking change, should my [version bump pr](https://github.com/InformedK12/email-repair/pull/17) go up to 1.0.0?

Reply:
Hmm, I don't think we ever specified which versions we support. Either way works for me.

Reply:
One approach some people take (which I don't necessarily advocate, but fwiw) is that pre `1.0` versions are wild west versions, so you can break as much as you want. Going `1.0` basically means it's "stable" and unlikely to change much after that. Of course, in terms of how much this is likely to change in the future, it's probably already `1.0` material anyway.